cmake/nng-config.cmake.in

```diff
@@ -8,9 +8,9 @@
 
 @PACKAGE_INIT@
 
-set(NNG_VERSION_MAJOR "@NNG_VERSION_MAJOR@")
-set(NNG_VERSION_MINOR "@NNG_VERSION_MINOR@")
-set(NNG_VERSION_PATCH "@NNG_VERSION_PATCH@")
+set(NNG_VERSION_MAJOR "@NNG_MAJOR_VERSION@")
```

Comment:
These haven't been populating properly for quite some time. I'm not sure if it's useful having these properties or not.

Reply:
LOl. :-)